tools-image/update-os-release.sh

```diff
@@ -17,7 +17,9 @@
 
 set -ex
 
+[ -f "/workspace/kairos-release" ] && sed -i -n '/KAIROS_/!p' /workspace/kairos-release
+# Clean up old os-release just in case so we dont have stuff lying around
 sed -i -n '/KAIROS_/!p' /workspace/os-release
-envsubst >>/workspace/os-release < /os-release.tmpl
+envsubst >>/workspace/kairos-release < /kairos-release.tmpl
```

Comment:
Don't we need the os-release fallback here too? What is this script used for? I can't find a consumer.

Reply:
our examples use it in the kairos repo. we should change those to use kairos-agent directly somehow and then we can deprecate this